VultisigApp/VultisigApp/View Models/VaultDetailViewModel.swift

```diff
@@ -11,6 +11,7 @@ import SwiftUI
 @MainActor
 class VaultDetailViewModel: ObservableObject {
     @Published var coinsGroupedByChains = [GroupedChain]()
+    @Published var selectedGroup: GroupedChain? = nil
```

Comment:
why we need published property? 

I think we can just have one computed property, that return array of actions

Reply:
Its there to fetch the Chain with both Send and Swap option. Just to keep the preset Coin consistent for both Send and Swap view.
Also, had to set this as a Published property since sometimes it'll try to search for a chain before the Categorization logic is finished and wont show the actions at all. Its there to rerender the component after that sorting it complete.